packages/ketcher-core/src/application/render/renderers/PolymerBondRenderer.md

```diff
@@ -0,0 +1,88 @@
+# Class: PolymerBondRenderer
+
+## How Snake Bonds Algorithm Works
+
+### Prerequisites:
+1) Only attachment points ['R1', 'R2'] have snake bond mode;
+2) Only Peptide and Chem monomers have snake bond mode;
+3) If there is a second monomer, then Snake bond mode could be enabled only when 2 monomers are not on the same horizontal line which means the distance of their Y-axis model coordinate is more than 0.5.
+4) There are still free attachment points for both monomers.
+5) There is no more than 1 bond between the first and the second monomer.
+6) Snake mode and Single Bond memu are enabled.
+
+### Workig Flow
+
+Events triggered from: `Bond.ts`
+
+1. When mousedown a monomer, the snake bond will start to be created. 
+Refer to `RenderersManager - addPolymerBond`
+2. When mousemove, the Snake Bonds Algorithm will be constantly called to calculate the position of the snake bond and move it.  
+Refer to `RenderersManager - movePolymerBond`
+3. When mouseup a monomer, the snake bond creation will be finished. 
+Refer to `RenderersManager - finishPolymerBondCreation`
+
+### Algorithm for Drawing Snake Bonds:
+
+It will first calculate the position of the second monomer relative to the first monomer, then draw the snake bond differently according to the result. 
+
+1) ![Alt text](../../../../docs/data/snake-bond-top-right.png)  
```

Comment:
Also, I propose to add move descriptive captions to image. For example, this is how I see it (not necessary to do it exactly this way):
![image](https://github.com/epam/ketcher/assets/41272128/c9a74fd2-b551-4682-909e-799c7f2745d0)


So i have an image on the left and I can:
1) Easily understand, which one is `Monomer 1` and `Monomer 2`
2) Which attachment point is used to create a bond from
3) Attachment points are switched (for some reason, may be code reusability, idk)
4) Numbers 1-2-3-4-5 represent the sequence in which bond is drawn.

Please, note: may be I understood the algorithm incorrectly, so my image can be wrong.